includes/admin/pages.php

```diff
@@ -273,73 +273,76 @@ public function html()
     	}
     	// It will display when the Pro is activated.
     	if (!is_plugin_active('Simple-Calendar-Google-Calendar-Pro-main/simple-calendar-google-calendar-pro.php')) { ?>
-								<div class="simcal-pt-[41px] simcal-max-w-[100%] simcal-ml-[5%]">
-									<div class="simcal-flex simcal-h-[430px] xl:simcal-h-[450px] ">
-										<div class="simcal-w-[48%] simcal-flex simcal-relative simcal-bg-sc_green-100 simcal-rounded-l-[5px] ">
-											<div class="simcal-pl-[37px] simcal-pt-[20px]">
-												<img src="<?php echo $admin_image_about_path . '/pro-banner.png'; ?>" />
```

Comment:
If this image is not being used anywhere then we can delete it.

Reply:
ok